### DISCUSSION TITLE
Create animation(.gif) by using the PyDPF.

### DISCUSSION THREAD

**Discussion by velpulagopiraju** [ansys/pydpf-core#1580] [General]
please let me know can i able to create the animation for "transient Fluid simulation problems". i have fluent project file(.flprj)
when i try to create an animation for 100 timesteps i just getting first time step frame only.. but i need to create the animation .gif file for the timesteps from 1to 100.
i have been use the time scoping as well but still not getting what i required.
Anyone please provide any solution for this.
![1](https://github.com/ansys/pydpf-core/assets/124872419/aa8e2c3e-5014-4bb6-8b1c-c290d53c57c6)

**Comment by PProfizi** [comment]
Hi @velpulagopiraju, 

Can you please try `mass_flow.animate()` instead of the `animate_mode` helper?

There is a [FieldsContainer.animate](https://dpf.docs.pyansys.com/version/stable/api/ansys.dpf.core.fields_container.html#ansys.dpf.core.fields_container.FieldsContainer.animate) method which does what you want.
It is showcased in [this example](https://dpf.docs.pyansys.com/version/stable/examples/06-plotting/06-animate_results.html).
